packages/melos/lib/src/common/workspace_changelog.dart

```diff
@@ -58,12 +58,14 @@ class WorkspaceChangelog {
         .where((update) => update.reason == PackageUpdateReason.graduate);
     final packagesWithBreakingChanges = pendingPackageUpdates.where(
       (update) =>
-          update.reason == PackageUpdateReason.commit &&
+          (update.reason == PackageUpdateReason.commit ||
+              update.reason == PackageUpdateReason.manual) &&
           update.semverReleaseType == SemverReleaseType.major,
```

Comment:
Based on https://github.com/invertase/melos/pull/242/files#diff-8a6755f20b64cdbfb7a9290c81f41a53b6b9d63fb4f7c60e1fa4641cce172572R145-R152 - manual updates would never be `SemverReleaseType.major` so is Is this change needed?

Reply:
Manually versioned packages can have commits, but if they don't, we still need a `SemverReleaseType`. That's what that check is for. If a manually versioned package has commits, some of them can be breaking and `MelosPendingPackageUpdate .semverReleaseType` would return `SemverReleaseType.major`.

Maybe a better approach would be to make `MelosPendingPackageUpdate.semverReleaseType` nullabe and return `null` for manually versioned packages. If `semverReleaseType` is `null` we determine whether to include the breaking change note in the changelog based on if there are any breaking change commits.